CODE_REVIEW_CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+* Be explicit. Remember people do not always understand your intentions online.
+* Be humble. ("I'm not sure - let's look it up.")
+* Do not use hyperbole. ("always", "never", "endlessly", "nothing")
+* Do not use sarcasm.
+* Keep it real. If emoji, animated gifs, or humor are not you, do not force them.
+  If they are, use them with aplomb.
+* Talk in person if there are too many "I did not understand" or "Alternative
+  solution:" comments. Post a follow-up comment summarizing offline discussion.
+
+Having Your Code Reviewed
+-------------------------
+
+* Document your pull request with details that provide context about the need for this change,
+  the nature of the change, and any related design considerations.
+* Document the level-of-formality of the code you are contributing or revising.  Is the
+  code intended to be an initial design?  Are you expecting major changes?  Is the code
+  design well-constrained by user needs?
+* Be grateful for the reviewer's suggestions. ("Good call. I'll make that
+  change.")
+* Do not take it personally. The review is of the code, not you.
+* Explain why the code exists. ("It's like that because of these reasons. Would
+  it be more clear if I rename this class/file/method/variable?")
+* Extract some changes and refactorings into future tickets/stories.
+* Link to the code review from the ticket/story. ("Ready for review:
+  https://github.com/organization/project/pull/1")
+* Push commits based on earlier rounds of feedback as isolated commits to the
+  branch. Do not squash until the branch is ready to merge. Reviewers should be
+  able to read individual updates based on their earlier feedback.
+* Seek to understand the reviewer's perspective.
+* Try to respond to every comment.
+* Wait to merge the branch until Continuous Integration tests
+  tells you the test suite is green in the branch.
+* Merge once you feel confident in the code and its impact on the project.
+
+Reviewing Code
+--------------
+
+Understand why the change is necessary (fixes a bug, improves the user
+experience, refactors the existing code). Then:
+
+* Communicate which ideas you feel strongly about and those you do not.
+* Identify ways to simplify the code while still solving the problem.
+* If discussions turn too philosophical or academic, move the discussion offline
+  to a regular project meeting. In the meantime, let the
+  author make the final decision on alternative implementations.
+* Offer alternative implementations, but assume the author already considered
+  them. ("What do you think about using a custom validator here?")
+* Seek to understand the author's perspective.
+* Plan to review the entire pull request.  Do not perform a partial review, and then add more comments later.
+* Sign off on the pull request with a :thumbsup: or "Ready to merge" comment.
+
+Style Comments
+--------------
+
+Pyomo currently does not have required style guidelines.  
```

Comment:
Maybe with the exception that docstrings be correctly parseable by ReadTheDocs?

Reply:
But there is a "Coding Standards" area in https://pyomo.readthedocs.io/en/stable/contribution_guide.html - which requires 4 space indents and desires PEP8 standards.